tests/Functional/Action/Security/LoginActionTest.php

```diff
@@ -22,20 +22,32 @@
 
 namespace App\Tests\Functional\Action\Security;
 
-use App\Tests\Traits\DatabaseFixturesTrait;
+use App\Tests\Traits\DatabaseTestingTrait;
+use Symfony\Bundle\FrameworkBundle\KernelBrowser;
```

Comment:
Same here, it looks like unneeded use statement

Reply:
Why it in unneded?